src/GraphQLParser/Location.cs

```diff
@@ -6,26 +7,20 @@ namespace GraphQLParser
     {
         private static readonly Regex _lineRegex = new Regex("\r\n|[\n\r]", RegexOptions.ECMAScript);
 
-        public Location(ISource source, int position)
-            : this(source.Body, position)
-        {
-           
-        }
-
-        public Location(string source, int position)
+        public Location(ReadOnlySpan<char> source, int position)
         {
             Line = 1;
             Column = position + 1;
 
             if (position > 0)
             {
-                var matches = _lineRegex.Matches(source);
+                var matches = _lineRegex.Matches(source.ToString()); // TODO: heap allocation
```

Comment:
I didn’t find the method I needed to run regexp againts ROM/Span.

Reply:
We should eliminate all regex.  It should be extremely easy, especially for this code!  Of course this is only used for error messages, but still...

Reply:
I know. I'm not ready to fix this code yet. In any case, the fix can be made without changing the public api in future releases. Published #102 .